### DISCUSSION TITLE
Could not load library libcublasLt.so.12. Error: libcublasLt.so.12: cannot open shared object file: No such file or directory

### DISCUSSION THREAD

**Discussion by charlinopen** [PaddlePaddle/PaddleOCR#13553] [Q&A]
按照README安装paddleocr 2.7.0.3, paddlepaddle-gpu 2.6.1。

运行环境如下：
GPU Compute Capability: 6.1
GPU Driver API Version: 12.4
CUDA Runtime API Version: 11.8
cuDNN Version: 8.9

执行： paddleocr --image_dir ./ch.jpg --rec false

报错： Could not load library libcublasLt.so.12

cuda版本的确是11.8， 故没有 12版本的库， 请问如何在cuda 11 环境下运行paddleocr？

**Comment by SWHL** [comment]
从问题来看，还是环境没有装好，需要再重新装一下。
也可以尝试使用Docker镜像，我找到一个之前的镜像：https://hub.docker.com/r/paddlecloud/paddleocr
